giskard/scanner/performance/metrics.py

```diff
@@ -52,10 +56,16 @@ def _calculate_metric(self, y_true: np.ndarray, y_pred: np.ndarray, model: BaseM
     def _calculate_affected_samples(self, y_true: np.ndarray, y_pred: np.ndarray, model: BaseModel) -> int:
         return len(y_true)
 
+    def _calculate_ctable_values(self, value, num_affected) -> list[list[int]]:
+        x = round(value * num_affected)
+        y = num_affected - x
+        return [x, y]
+
 
 class Accuracy(ClassificationPerformanceMetric):
     name = "Accuracy"
     greater_is_better = True
+    has_contingency_table = True
```

Comment:
Same as above, this could be more general. In any case, I’m not sure it's needed if we have it on the result object.

Reply:
It's not needed but I added it for clarity and "just in case". Should I remove it?

Reply:
Actually, it's needed because it's used to decide whether to calculate the binary counts (to prevent from calculating them for classification metrics where it wouldn't make sense, e.g. the F1-Score)